src/applications/search/actions/index.js

```diff
@@ -17,7 +17,7 @@ export function fetchSearchResults(query, page, analyticsMetaInfo) {
 
     return apiRequest(queryString)
       .then(response => {
-        if (analyticsMetaInfo) {
+        if (analyticsMetaInfo && (page === 1 || page === undefined)) {
```

Comment:
This feature is to prevent repeated event loggings when a user navigates to a later page of results for the same query (only log an event if we are pulling the 1st page, or if the page has not been set yet)